docs/source/guide/Docker_README.md

```diff
@@ -73,13 +73,30 @@ Example: `redis://redis`
 
 #### (Required) `RSTUF_SQL_SERVER`
 
-SQL server address.
+RSTUF requires [PostgreSQL](https://www.postgresql.org).
 
-The SQL Server must to be compatible with
-[SQLAlchemy](https://www.sqlalchemy.org). RSTUF recomends
-[PostgreSQL](https://www.postgresql.org).
```

Comment:
By removing those and replacing them with line 76 we insist that we accept only `PostgresSQL`, right?

Reply:
Yes, correct.
Let's start by suggesting a database we have tested and used.